<launch>
  <arg name="filter_config" default="$(find dbot_ros)/config/gaussian_tracker.yaml"/>
  <arg name="camera_config" default="$(find dbot_ros)/config/camera.yaml"/>
  <arg name="object_config" default="$(find dbot_ros)/config/object.yaml"/>

  <node 
    name="gaussian_tracker" 
    type="gaussian_tracker"
    pkg="dbot_ros" 
    output="screen" 
    respawn="false"
    launch-prefix="">
        <rosparam command="load" file="$(arg filter_config)" />
        <rosparam command="load" file="$(arg camera_config)" />
        <rosparam command="load" file="$(arg object_config)" />
        <!-- <param name="config_file"  type="string"  value="$(arg config_file)" /> -->
  </node>
</launch>
